solvebio/resource/dataset.py

```diff
@@ -22,75 +22,61 @@ class Dataset(CreateableAPIResource,
               UpdateableAPIResource):
     """
     Datasets are access points to data. Dataset names are unique
-    within versions of a depository.
+    within a vault folder.
     """
+    USES_V2_ENDPOINT = True
```

Comment:
@jsh2134, good idea.  @davecap, these flags will not be removed as they indicate to the client to use the /v2 endpoints.  That other flag, "force_use_v1", will be removed prior to release.